src/to_wkt/mod.rs

```diff
@@ -8,6 +8,18 @@ pub use geo_trait_impl::{
     write_rect, write_triangle,
 };
 
+/// A wrapper around something that implements std::io::Write to be used with our writer traits,
+/// which require std::fmt::Write
+struct WriterWrapper<W: std::io::Write>(W);
+
+impl<W: std::io::Write> std::fmt::Write for WriterWrapper<W> {
+    fn write_str(&mut self, s: &str) -> std::fmt::Result {
+        // Sadly, this will lose the content of the error when mapping to std::fmt::Error
+        self.0.write(s.as_bytes()).map_err(|_| std::fmt::Error)?;
```

Comment:
This loss of error content is unfortunate but I'm not sure of a good way around this.

Reply:
See #129